cloud-native-archival/permissions-group-BASIC/v2.json

```diff
@@ -0,0 +1,122 @@
+        "use_case": "Use to store the archived snapshots in a blob container in a storage account.",
+        "scope": "resourceGroup"
+      },
+      {
+        "value": "Microsoft.Storage/storageAccounts/blobServices/containers/write",
+        "use_case": "Creating writing and deleting page blob containers during the cross-region replication.",
+        "scope": "resourceGroup"
+      },
+      {
+        "value": "Microsoft.Storage/storageAccounts/blobServices/containers/delete",
+        "use_case": "Creating writing and deleting page blob containers during the cross-region replication.",
+        "scope": "resourceGroup"
+      },
+      {
+        "value": "Microsoft.Resources/subscriptions/resourceGroups/read",
+        "use_case": "Use to create the managed identity storage account and container for a resource.",
+        "scope": "resourceGroup"
+      },
+      {
+        "value": "Microsoft.Resources/subscriptions/resourceGroups/write",
+        "use_case": "Creating a separate resource group to store snapshots taken by RSC.",
+        "scope": "resourceGroup"
+      },
+      {
+        "value": "Microsoft.ManagedIdentity/userAssignedIdentities/assign/action",
+        "use_case": "Use to assign the user-assigned managed identity to the storage account to encrypt the data using server-side encryption and customer-managed key.",
+        "scope": "resourceGroup"
+      },
+      {
+        "value": "Microsoft.Storage/storageAccounts/encryptionScopes/read",
+        "use_case": "Use to create an encryption scope at the container level in a storage account created by Rubrik to store the archived snapshots.",
+        "scope": "resourceGroup"
+      },
+      {
+        "value": "Microsoft.Storage/storageAccounts/encryptionScopes/write",
+        "use_case": "Use to create an encryption scope at the container level in a storage account created by Rubrik to store the archived snapshots.",
+        "scope": "resourceGroup"
+      },
+      {
+        "value": "Microsoft.KeyVault/vaults/read",
+        "use_case": "Use to list all the Key Vaults available in a region to choose the key.",
+        "scope": "resourceGroup"
+      },
+      {
+        "value": "Microsoft.KeyVault/vaults/keys/read",
+        "use_case": "Use to list the keys in a Key Vault to choose the key for encrypting the data using serverside encryption and customer-managed key.",
+        "scope": "resourceGroup"
+      },
+      {
+        "value": "Microsoft.Storage/storageAccounts/blobServices/read",
+        "use_case": "Use to enable versioning on storage accounts and reading blob service properties of a storage account.",
+        "scope": "resourceGroup"
+      },
+      {
+        "value": "Microsoft.Storage/storageAccounts/blobServices/write",
+        "use_case": "Enabling versioning on storage accounts and reading blob service properties of a storage account.",
+        "scope": "resourceGroup"
+      },
+      // Added in version 2
```

Comment:
As JSON doesn't natively support comments, I don't think we should include them in this file.

Reply:
Nice catch!